From 7d2cae3064bb5a03fbe6c7dcc2068b6bfbac6cb4 Mon Sep 17 00:00:00 2001
From: "Guoniu.zhou" <guoniu.zhou@nxp.com>
Date: Tue, 18 May 2021 11:02:15 +0800
Subject: [PATCH 26/27] LF-2670-02: media: isi: core: disable gate clock for
 isi by default

Disable gate clock for ISI by default

Signed-off-by: Guoniu.zhou <guoniu.zhou@nxp.com>
---
 drivers/staging/media/imx/imx8-isi-core.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/staging/media/imx/imx8-isi-core.c b/drivers/staging/media/imx/imx8-isi-core.c
index 37d20f1d9..f6922fe25 100644
--- a/drivers/staging/media/imx/imx8-isi-core.c
+++ b/drivers/staging/media/imx/imx8-isi-core.c
@@ -773,6 +773,7 @@ static int mxc_isi_probe(struct platform_device *pdev)
 		dev_warn(dev, "Populate child platform device fail\n");
 
 	mxc_isi_clk_disable(mxc_isi);
+	disp_mix_clks_enable(mxc_isi, false);
 
 	platform_set_drvdata(pdev, mxc_isi);
 	pm_runtime_enable(dev);
-- 
2.32.0

